views/incubator/omniledger.tpl

```diff
@@ -0,0 +1,101 @@
+<h2>Blockchains</h2>
+
+<p>The hype of blockchains is still ongoing, and so it is important to understand where
+    and how they can actually solve problems.
+    The most important question to ask before using a blockchain is the following:<br>
+    <i>Is there a single trusted legal entity to solve my problem?</i><br>
+    If the answer to this question is <strong>YES</strong>, then you don't need a blockchain.
+    An example is a bank account, where you do trust your bank to hold your money.
+    Or health certificates, where you trust the state to hand them out correctly.
+    On the other hand, if you don't want or cannot trust a single entity with your problem, then a blockchain
+    can be a good solution.
+    Examples of this include money transfers if you don't trust any bank, or authentication to services
+    without trusting only one (like Google or Facebook).<br>
+    Three important elements of blockchains are:
+</p>
+
+<table border="0">
+    <tr style="vertical-align: top;">
+        <td width="33%" style="padding: 1em;">
+            <h3>Decentralization</h3>
+            <img src="/resources/incubator/images/omniledger/decentralized.png" width="33%"
+                 style="float: left; padding: 1em;"/>
+            Blockchains got a lot of publicity under the name of <strong>Bitcoin</strong>.
+            For some it is a source of great wealth, for others it is a synonym for a lot
+            of wasted energy. Few remember the origin of Bitcoin, which is about
+            <strong>decentralization</strong>.
+            Decentralization means that there is no central entity, but that the responsibility
+            is spread out over several entities. In the example of Switzerland, one could
+            say that the organization in cantons leads to a decentralized government.
+        </td>
+        <td width="33%" style="padding: 1em;">
+            <h3>Transactions</h3>
+            <img src="/resources/incubator/images/omniledger/transaction_general.png" width="33%"
+                 style="float: left; padding: 1em;"/>
+            <img src="/resources/incubator/images/omniledger/transaction_money.png" width="33%"
+                 style="float: right; padding: 1em;"/>
+            A transaction is what is stored in a blockchain.
```

Comment:
In my browser, the "A" is actually rendered on top of the images.

Reply:
Same here :( Fixed now.